pkg/meta/base.go

```diff
@@ -222,7 +222,7 @@ func (m *baseMeta) CleanStaleSessions() {
 		return
 	}
 	for _, sid := range sids {
-		m.en.doCleanStaleSession(sid)
+		logger.Infof("clean up stale session %d: %s", sid, m.en.doCleanStaleSession(sid))
```

Comment:
only log the error?

Reply:
We need it to find out the reason why a session is removed. Besides, this log won't be many because client calls CloseSession to gracefully exit.